generator/graphql-kotlin-federation/src/test/kotlin/com/expediagroup/graphql/generator/federation/data/queries/federated/Product.kt

```diff
@@ -124,6 +124,19 @@ data class User(
     @ExternalDirective val name: String
 )
 
+/*
+type Author {
```

Comment:
based on the code the type is actually

```graphql
type Author @extends @key(fields: "authorId") {
  authorId: Int! @external
  name: String: @external
```

If both fields are external -> should it be `@key(fields: "authorId", resolvable: false)`? or should there be at least a single local field?